src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+
+        return BallColor.values()[random.nextInt(BallColor.values().length)].name();
```

Comment:
It would be easier to read it if you split it into two lines
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].name();
```

Reply:
got it